pkg/test/istio.io/README.md

```diff
@@ -1,13 +1,42 @@
-# Istio.io Example Tests
+# Istio.io preliminary test cases
+
+Welcome to the development of the istio.io preliminary test cases!
+
+This work is experimental. This work is also under construction and may not work
+for you. The Istio developers have done some work to document known problems or
+workarounds. We expect this work will take several releases to complete.
 
 ## Overview
-This folder contains tests for the different examples on Istio.io.
+This folder contains tests for the different examples on [istio.io](https://istio.io).
+The goal of this work is to ensure [istio.io](https://istio.io) is tested according
+to the code in the [Istio repository](https://github.com/istio).
 
 ## Writing Tests
 
+This section needs fleshing out.
 
 ## Executing Tests Locally using Minikube
-* Create a Minikube environment as advised on istio.io
-* Run the tests as follows:
- go test ./... -p 1 --istio.test.env kube --istio.test.kube.config ~/.kube/config -v
 
+- Create a [Minikube environment](https://preliminary.istio.io/docs/setup/kubernetes/platform-setup/minikube/)
+  to develop the code.
+
+There has been some indication that the default minikube environment values of 16GB
+and 4 cores is not sufficient to run the test cases.  If after running the test cases,
+you see istio-system pods in the pending state, please attempt to debug the reason
+why they are pending so the Istio developers can improve these test cases.  Avaialbity
+of running the test cases locally is crucial to the progression of this work.
+
+- Change into the proper directory to run the test cases
+
+```bash
+$ cd $HOME/istio/pkg/test/istio.io
+```
+
+- Run the istio.io test cases:
+
+```bash
+$ go test ./... -p 1 --istio.test.env kube --istio.test.kube.config ~/.kube/config -v
```

Comment:
nit - `~/.kube/config` is default afaik

Reply:
HUB and TAG are required as well. I personally always use:

```
export HUB="gcr.io/istio-release"
export TAG="master-latest-daily"
```

Reply:
thanks, I will list this as well.  The developer wiki that was written ages ago recommends:
```
# Please change HUB to the desired HUB for custom docker container
# builds.
export HUB="docker.io/$USER"

# The Istio Docker build system will build images with a tag composed of
# $USER and timestamp. The codebase doesn't consistently use the same timestamp
# tag. To simplify development the development process when later using
# updateVersion.sh you may find it helpful to set TAG to something consistent
# such as $USER.
export TAG=$USER

# If your github username is not the same as your local user name (saved in the
# shell variable $USER), then replace "$USER" below with your github username
export GITHUB_USER=$USER
```

Reply:
I personally still have that HUB, TAG set per the docs.